docs/source/model-registry.rst

```diff
@@ -152,6 +152,57 @@ While the method above creates an empty registered model with no version associa
         run_id="d16076a3ec534311817565e6527539c0"
     )
 
+Run a MLflow Model from the Model Registry
+------------------------------------------
+
+After you have registered an MLflow model, you can run that model using :meth:`~mlflow.sklearn.load_model`. 
+
+**Run a specific model version**
+
+To fetch a specific model version, just supply that version name as part of the model URI.
+
+.. code-block:: py
+
+    import mlflow.sklearn
+
+    model_name = "sk-learn-random-forest-reg-model"
+    model_version = 1
+
+    model = mlflow.sklearn.load_model(
+        model_uri=f"models:/{model_name}/{model_version}
+    )
+
+**Run the latest model version in a specific stage**
+
+Model Registry allows you to transition multiple versions of the model into the same or different stages. To fetch the model in its stage name, simply provide the model stage label as part of the model URI, and it will fetch the most recent version of the model in that stage.
```

Comment:
```suggestion
To fetch a model version by stage, simply provide the model stage as part of the model URI, and it will fetch the most recent version of the model in that stage.
```